federatorai/README.md

```diff
@@ -0,0 +1,147 @@
+# Federator.ai Integration
+
+## Overview
+
+This check monitors [ProphetStor Federator.ai][1].
+
+ProphetStor Federator.ai is an AI-based solution that helps enterprise manage, optimize, auto-scale resources for any applications on Kubernetes. Using advanced machine learning algorithms to predict application workload, Federator.ai scales the right amount of resources at the right time for optimized application performance.
+
+* AI-based workload prediction for Kafka or any applications
+* Resource recommendation based on workload prediction, application, Kubernetes and other related metrics
+* Automatic scaling of application containers through [Datadog Watermark Pod Autoscaler (WPA)][4]
+
```

Comment:
The above paragraphs explain what Federator.ai is, but we'd like to show users what is the value they get from this integration. 

Please consider to add a paragraph to the Overview section that explains what users will get from this integration. i.e. dashboards that visualize the predictions and the recommendation in timeseries, and alongside how the application (like Kafka) performs